cmd/utils/flags.go

```diff
@@ -513,6 +520,36 @@ func MakeAccountManager(ctx *cli.Context) *accounts.Manager {
 	return accounts.NewManager(ks)
 }
 
+// MakeExtra resolves extradata for the miner from a flag or returns a default
+func MakeExtra(ctx *cli.Context) []byte {
```

Comment:
I don't think there's a particular reason to expose this function to the public API. Many others were exposed because they were used outside of the `utils` package, but if this one is not, then I'd say let's keep it private and expose if need be.